docs/lifetime/index.rst

```diff
@@ -21,35 +62,77 @@ As you work in your application, it's good to remember these concepts so you mak
 
 - You have a global singleton logging service.
 - Two simultaneous requests come in to the web application.
 - Each request is a logical "unit of work" and each requires its own order processing service.
-- Each order processing service needs to log information to the logging service.
+- Each controller needs to log information to the logging service.
+
+In this scenario, you'd have a root lifetime scope that contains the singleton logging service and you'd have one child lifetime scope per request, each with its own controller instance.
+
+From a class registration standpoint, it might look like this:
+
+.. sourcecode:: csharp
+
+    var builder = new ContainerBuilder();
+
+    // The logger is a singleton - shared everywhere.
+    builder.RegisterType<Logger>()
+           .As<ILogger>()
+           .SingleInstance();
+
+    // Controllers are created per lifetime scope. They
+    // take an ILogger as a parameter.
+    builder.RegisterType<Controller>()
+           .InstancePerLifetimeScope();
 
-In this scenario, you'd have a root lifetime scope that contains the singleton logging service and you'd have one child lifetime scope per request, each with its own order processing service::
+    var container = builder.Build();
 
-    +---------------------------------------------------+
-    |                 Autofac Container                 |
-    |                Root Lifetime Scope                |
-    |                                                   |
-    |                  Logging Service                  |
-    |            (shared across all requests)           |
-    |                                                   |
-    | +----------------------+ +----------------------+ |
-    | |  First Request Scope | | Second Request Scope | |
-    | |                      | |                      | |
-    | |   Order Processor    | |   Order Processor    | |
-    | +----------------------+ +----------------------+ |
-    +---------------------------------------------------+
+As web requests come in, lifetime scopes might look like this:
 
-When each request ends, the request lifetime scope ends and the respective order processor gets disposed. The logging service, as a singleton, stays alive for sharing by future requests.
+.. image:: lifetime-scope-web-example.png
 
-You can dive deeper on lifetime scopes in `Nick Blumhardt's Autofac lifetime primer <http://nblumhardt.com/2011/01/an-autofac-lifetime-primer/>`_.
+A rough sequence of events for a web app framework like :doc:`ASP.NET Core <../integration/aspnetcore>` goes like this:
 
+1. When a web request comes in, the web application framework creates a child lifetime scope - the "request lifetime scope."
+2. The web application framework resolves the controller instance from the request lifetime scope. Since the controller is registered as "instance per lifetime scope," it will be a sort of "singleton" within that request but will not be shared with other requests.
```

Comment:
It may be a bit risky to mix the use of the word "singleton" in this section, even if it is couched with "sort of". 

(also, comma misplaced)

```suggestion
2. The web application framework resolves the controller instance from the request lifetime scope. Since the controller is registered as "instance per lifetime scope", this instance will be shared by any components within that request but will not be shared with other requests.
```

Reply:
Updated phrasing, but I think the comma thing is [US English vs. UK English](https://www.grammarly.com/blog/quotation-marks/) - I'll stand by the placement on that one.

Reply:
US English has commas inside the quote? You guys are bonkers.

Reply:
You have _no idea_. Oh, wait, you've probably watched politics for the last few years, you _totally have an idea_. 🤪

Reply:
This is the last straw; it was always going to be something to do with grammar.